content/en/guide/v10/typescript.md

```diff
@@ -258,72 +331,7 @@ const AppContext = createContext({
 // }>
 ```
 
-It also requires you to pass in all the properties you defined in the initial value:
-
-```tsx
-function App() {
-  // This one errors 💥 as we haven't defined theme
-  return (
-    <AppContext.Provider
-      value={{
-//    ~~~~~ 
-// 💥 Error: theme not defined
-        lang: "de",
-        authenticated: true
-      }}
-    >
-    {}
-      <ComponentThatUsesAppContext />
-    </AppContext.Provider>
-  );
-}
-```
-
-If you don't want to specify all properties, you can either merge default values with overrides:
-
-```tsx
-const AppContext = createContext(appContextDefault);
-
-function App() {
-  return (
-    <AppContext.Provider
-      value={{
-        lang: "de",
-        ...appContextDefault
-      }}
-    >
-      <ComponentThatUsesAppContext />
-    </AppContext.Provider>
-  );
-}
-```
```

Comment:
This is mere explanation about how to pass values to `props` using destructuring assignment.
I think it is not suitable in this page.
Therefore, I removed this sentence.